src/sys/windows/tcp.rs

```diff
@@ -64,3 +64,8 @@ pub fn bind(addr: SocketAddr) -> io::Result<net::TcpListener> {
         .map(|_| unsafe { net::TcpListener::from_raw_socket(socket as StdSocket) })
     })
 }
+
+pub fn accept(listener: &net::TcpListener) -> io::Result<(net::TcpStream, SocketAddr)> {
+    // The non-blocking state of `listener` is inherited.
```

Comment:
Is the source for this assumption the [Remark](https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-accept#remarks)? I don't think the source needs to be an explicit comment, but just want to make sure this comment isn't assuming from something else.

Reply:
I was indeed going from those docs, I've added a link in the last commit anyway. Always useful for people (and future me when I forget).